core/common/src/thrift/block_worker.thrift

```diff
@@ -122,15 +110,4 @@ service BlockWorkerClientService extends common.AlluxioService {
     /** deprecated since 1.3.0 and will be removed in 2.0 */ 2: list<i64> metrics,
     )
     throws (1: exception.AlluxioTException e)
-
-  /**
-   * Used to unlock a block after the block is accessed, if the block is to be removed, delete the
-   * block file. return true if successfully unlock the block, return false if the block is not
-   * found or failed to delete the block.
-   */
-  bool unlockBlock(
```

Comment:
why we remove `lockBlock` and `unlockBlock`, while leaving other methods (but return `UnsupportedException`)?